### ISSUE TITLE
In 0.6.0, the image tag expected v0.6.0, but in reality 0.6.0 is used.

### ISSUE THREAD

**Issue by kahirokunn** [bitpoke/mysql-operator#779]

**Comment by enoliveira** [comment]
```
Failed to pull image "docker.io/bitpoke/mysql-operator-orchestrator:0.6.0": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/bitpoke/mysql-operator-orchestrator:0.6.0": failed to resolve reference "docker.io/bitpoke/mysql-operator-orchestrator:0.6.0": docker.io/bitpoke/mysql-operator-orchestrator:0.6.0: not found
```

**Comment by calind** [comment]
This is fixed in [v0.6.1](https://github.com/bitpoke/mysql-operator/releases/tag/v0.6.1). Thanks for reporting!
